yarn.lock

```diff
@@ -9990,9 +9990,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#72b5c131b4a9ebd6b5773722ce8d923a86196ff2":
-  version "3.44.0"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#72b5c131b4a9ebd6b5773722ce8d923a86196ff2"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#86263a396bfe2b13153797bac78c424a913b450d":
+  version "3.46.0"
```

Comment:
3.45 was some validation updates that Rian pushed for another form. I've confirmed with him that we're good to skip up to 3.46, which contains these servicePeriod changes